realsense2_camera/src/base_realsense_node.cpp

```diff
@@ -93,10 +93,10 @@ BaseRealSenseNode::BaseRealSenseNode(ros::NodeHandle& nodeHandle,
     _stream_name[RS2_STREAM_DEPTH] = "depth";
     _depth_aligned_encoding[RS2_STREAM_DEPTH] = sensor_msgs::image_encodings::TYPE_16UC1;
 
-    // Infrared stream
-    _image_format[RS2_STREAM_INFRARED] = CV_8UC1;    // CVBridge type
-    _encoding[RS2_STREAM_INFRARED] = sensor_msgs::image_encodings::MONO8; // ROS message type
-    _unit_step_size[RS2_STREAM_INFRARED] = sizeof(uint8_t); // sensor_msgs::ImagePtr row step size
+    // Infrared-rgb stream
```

Comment:
Changing the infrared stream to to be 3 channel (color) is causing several [tests](https://travis-ci.org/plusone-robotics/realsense/builds/514801070#L5101) to fail.  An alternative approach would be to create a second stream OR add a "switch" to enable colorized IR.  By default it would run IR only, allowing the failing tests to pass.

Reply:
that is the intent for further commits
I could add another stream, but I feel that would have potential problems down the line
I like the idea to add a switch to change the IR to the RGB